virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc

```diff
@@ -18,19 +18,31 @@ To attach a virtual machine to an additional network:
 The network attachment definition must be in the same namespace as the pod or virtual machine.
 ====
 . Attach the virtual machine to the network attachment definition by using either the web console or the CLI:
+. Configure a Linux bridge.
+. Configure a `NetworkAttachmentDefinition` for a namespace in the web console or CLI.
+. Attach the virtual machine to the `NetworkAttachmentDefinition` by using either the web console or the CLI:
 ** In the web console, create a NIC for a new or existing virtual machine.
 ** In the CLI, include the network information in the virtual machine configuration.
 
+[NOTE]
+====
+There are multiple methods for configuring a VLAN, including `NetworkAttachmentDefinition` and `NodeNetworkConfigurationPolicy`.
+However, a `NetworkAttachmentDefinition` provides a more efficient and more manageable configuration.
+====
+
 include::modules/virt-networking-glossary.adoc[leveloffset=+1]
 
-[id="virt-creating-network-attachment-definition"]
-== Creating a network attachment definition
+[id="virt-creating-linux-bridge"]
+== Configuring a Linux bridge
```

Comment:
What is the reasoning behind including this entire module here instead of keeping the Prerequisites section as is with a link to the node networking page?